Empire/Empire.cpp

```diff
@@ -1458,6 +1498,8 @@ int Empire::StockpileID(ResourceType res) const {
         return m_capital_id;
         break;
     case RE_INDUSTRY:
+        return IMPERIAL_OBJECT_ID;
```

Comment:
Ok. Question: Should I switch to an empire meter instead using the resource pool stockpile as well?